gcp-jobs/future-effective-filings/src/future_effective_filings/file_future_effective.py

```diff
@@ -0,0 +1,116 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Future Effective Date service.
+
+This module script is for putting filings with future effective dates on the entity filer queue.
+"""
+import asyncio
+import os
+import uuid
+from datetime import UTC, datetime
+
+import config  # pylint: disable=import-error
+import requests
+from dotenv import find_dotenv, load_dotenv
+from flask import Flask
+from flask_api import status
+from gcp_queue import GcpQueue
+from simple_cloudevent import SimpleCloudEvent, to_queue_message
+
+from structured_logging import StructuredLogging
+
+DEFAULT_CONNECT_TIMEOUT = 2
+gcp_queue = GcpQueue()
+
+# this will load all the envars from a .env file located in the project root
+load_dotenv(find_dotenv())
+run_mode = os.getenv("FLASK_ENV", "production")
+
+def create_app(run_mode=run_mode):
+    """Return a configured Flask App using the Factory method."""
+    app = Flask(__name__)
+    app.config.from_object(config.CONFIGURATION[run_mode])
+    gcp_queue.init_app(app)
+    StructuredLogging().init_app(app)
```

Comment:
Can you initialize this the same as me here and use current_app for logging instead of a new logger in each file? https://github.com/bcgov/lear/pull/3383/files#diff-77df499b905d0664774e9fc54014094843a730ecc664484eebf47a5569a9f158

We're planning changes to the structured_logging that'll add more context to our logs and expect this type of initialization

Reply:
Already made that change yesterday so that one is ready to go